years/2019/01/solutions.ts

```diff
@@ -0,0 +1,16 @@
+export const part1 = (input: string[]): number => {
+  return input.reduce((acc: number, value: string) => (acc += calcuation(parseInt(value))), 0)
```

Comment:
Think it's nice to explicitly pass the radix param to `parseInt`:

<img width="535" alt="image" src="https://user-images.githubusercontent.com/24367010/228903040-ece3326c-9eaf-4d41-9c73-a4154ec076be.png">

```suggestion
  return input.reduce((acc: number, value: string) => (acc += calcuation(parseInt(value, 10))), 0)
```

I would normally reach for `Number` anyway, but it does the same job

```suggestion
  return input.reduce((acc: number, value: string) => (acc += calcuation(Number(value))), 0)
```

Reply:
TIL about number, it's probably more idiomatic, right?

Reply:
Probably yeah